src/index.jsx

```diff
@@ -24,6 +24,22 @@ function Index() {
   const [isAdmin, setIsAdmin] = useState(null);
 
   useEffect(() => {
+    registerSW({
```

Comment:
this is just a baseline, we would have to decide on what to do here, integrate and update interval, remove logging, extend it?

Reply:
have 2-3 issues worth of ideas here, will ping back later